doc/docgen.rst

```diff
@@ -14,12 +14,22 @@ Introduction
 ============
 
 This document describes the `documentation generation tools`:idx: built into
-the `Nim compiler <nimc.html>`_, which can generate HTML and JSON output
-from input .nim files and projects, as well as HTML and LaTeX from input RST
+the `Nim compiler <nimc.html>`_, which can generate HTML, Latex and JSON output
+from input ``.nim`` files and projects, as well as HTML and LaTeX from input RST
 (reStructuredText) files. The output documentation will include the module
 dependencies (`import`), any top-level documentation comments (`##`), and
 exported symbols (`*`), including procedures, types, and variables.
 
+===================   ======================   ============   ==============
+command               runs on...               input format   output format
+===================   ======================   ============   ==============
+`nim doc`:cmd:        documentation comments   ``.nim``       ``.html`` HTML
+`nim doc2tex`:cmd:    \`                       \`             ``.tex`` LaTeX
```

Comment:
how about `-` instead of backtick ?

Reply:
I don't know what `-` means. While \` aka "ditto" means the same. I just don't want repeat "documentation comments" 3 times.

Reply:
i didn't know about backtick had meaning of ditto, is that standard? anyways, I don't care too much either way

Reply:
it seems I confused something. Replaced to proper unicode ditto.